src/modules/auth/auth.service.ts

```diff
@@ -0,0 +1,50 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { RegisterDto } from './dto/request/register.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Auth } from './entities/auth.entity';
+import { Hasher } from 'src/shared/libs/hasher.lib';
```

Comment:
Fix the import path here as well.

Reply:
or you can configure paths in tsconfig.json